src/backend/cdb/motion/ic_udpifc.c

```diff
@@ -1175,6 +1175,7 @@ setupUDPListeningSocket(int *listenerSocketFd, uint16 *listenerPort, int *txFami
 	struct addrinfo 		*addrs = NULL;
 	struct addrinfo 		*addr;
 	struct addrinfo 		hints;
+	char					service[32];
```

Comment:
Can we get rid of the magic number? And replace it with more meaningful const or define?

Reply:
I'm ok if it is done in a separate change.